Toolchain/Patches/gcc.patch

```diff
@@ -363,7 +363,15 @@ index 326a279c5..6b76901dc 100755
  if test x$gcc_no_link = xyes; then
    # Setting cross_compile will disable run tests; it will
    # also disable AC_CHECK_FILE but that's generally
-@@ -29270,6 +29262,5986 @@ else
+@@ -29264,6 +29256,5 @@ if
+ 
+     $as_echo "#define HAVE_ICONV 1" >>confdefs.h
+ 
+-    $as_echo "#define HAVE_MEMALIGN 1" >>confdefs.h
+ 
+   else
+ 
+@@ -29270,6 +29261,5986 @@ else
```

Comment:
Ah, I thought I removed all references to iconv (and I noticed the typo myself too, but as I mentioned, I thought I removed it), sorry. As for `HAVE_ICONV`, I see we have headers for it but no implementations? I can remove it in my patch, just tell me how you want it.

Reply:
Well, I just changed the commit message for now and left removing ICONV for later (or somebody else?) because I don't feel like rebuilding the toolchain right now and it doesn't seems like it breaks anything right now, really.